lib/ReadsUtils/ReadsUtilsImpl.py

```diff
@@ -823,15 +839,75 @@ def upload_reads(self, ctx, params):
         self.log('Starting upload reads, parsing args')
```

Comment:
> I don't think we should delete the 'ctx' variable
Why not?
> (might have side effects?).
It can't possibly.

The only way I know of to silence this type of pep8 warning is to delete it or use it in some way, and deletion is the cleanest I've found. I do it in other modules without issues.